Makefile.am

```diff
@@ -118,6 +122,9 @@ util_fi_pingpong_LDADD = $(linkback)
 
 nodist_src_libfabric_la_SOURCES =
 src_libfabric_la_SOURCES =			\
+	include/ofi_hmem.h			\
+	include/ofi_hmem_cuda.h			\
+	include/ofi_hmem_system.h		\
```

Comment:
Please merge these into a single header file.  Each one only has 2 lines of code.

We can probably merge the .c files together as well.  We're adding nearly as many lines of license than code.

Reply:
That's on me. I asked him to separate the c files for easier future additions since there will be more device ops to add in the future (get handle, open, handle, close handle, ipc memcpy, init, cleanup) so eventually each set of device ops will be much longer than this set.

Reply:
So, instead of 2 lines per file, we might get up to 8?  :)  I'd still combine these, especially since the definitions will be in sync.

Reply:
lol no - specifically separating the c files

Reply:
So merge headers but keep C files?

Reply:
At the very least kill hmem_system.c.

Reply:
Done. Kept hmem_cuda.c.